workbench/config/WorkbenchConfig.php

```diff
@@ -53,6 +53,8 @@ function __construct() {
         $configNamespace = "forceworkbench";
         $configDelim = "__";
         foreach ($_ENV as $envKey => $envValue) {
+
```

Comment:
@rvwatch - Could we minimize unnecessary whitespace changes to the file? Thsi makes the diff very hard to follow.

Reply:
Is there a linter or approved php formatter that I can use so that I can keep this from happening? You gotta believe me when I say I'm not trying to add these... lol